src/models/ModelBuilder.cc

```diff
@@ -16,13 +16,11 @@ void ModelBuilder::processSubstitutions(MapWithKeys<PTRef,PtAsgn,PTRefHash> cons
             continue;
         }
         assert(logic.isVar(key));
-        PtAsgn target = copy[key];
-        if (target.sgn == l_True) {
-            PTRef mappedTerm = target.tr;
-            PTRef val = model->evaluate(mappedTerm);
-            assert(logic.isConstant(val));
-            assignCopy.insert(std::make_pair(key, val));
-        }
+        PTRef target = copy[key];
+        PTRef mappedTerm = target;
```

Comment:
Now we only need one of `target` and `mappedTerm`, the other can go away.

Reply:
Fixed.